sdk/python/feast/feature_store.py

```diff
@@ -1255,6 +1255,24 @@ def materialize_incremental(
         feature_views_to_materialize = self._get_feature_views_to_materialize(
             feature_views
         )
+
```

Comment:
Only change required here is, in apply(), when calling `_get_provider().update_infra()` check for `getattr(self.config.online_store, "lazy_table_creation", False)`. You can revert the rest of the changes and move changes to materialize_single_feature_view() in PassthroughProvider class.